lib/adapters/class-adapter.php

```diff
@@ -236,6 +275,15 @@ protected function get_searchable_post_types( array $post_types ): array {
 		return apply_filters( 'elasticsearch_extensions_searchable_post_types', $post_types );
 	}
 
+	/**
+	 * Returns whether a REST API search handler for suggestions can be enabled.
```

Comment:
```suggestion
	 * Returns whether a REST API search handler for suggestions is enabled.
```

Reply:
I see what you mean that "can be enabled" is somewhat awkward. But "is enabled" doesn't seem right to me, either, since the method doesn't check whether the handler is actually registered. For some requests, it might not be expected for REST search handlers to ever be registered.

I'm going with this for now: `Returns whether support for a REST API search handler for suggestions is enabled.` With that, "support" is what's checked, not the handler itself.